.github/workflows/preview.yml

```diff
@@ -0,0 +1,40 @@
+name: preview
+on: pull_request
+
+jobs:
+  update:
+    name: EAS Update
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      pull-requests: write
+    steps:
+      - name: Check for EXPO_TOKEN
+        run: |
+          if [ -z "${{ secrets.EXPO_TOKEN }}" ]; then
+            echo "You must provide an EXPO_TOKEN secret linked to this project's Expo account in this repo's secrets. Learn more: https://docs.expo.dev/eas-update/github-actions"
+            exit 1
+          fi
+
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18.x
+          cache: npm
```

Comment:
Since we have a `.nvmrc` file in the project where we define the version of node we use, we should tell the actions to use the exact same version by pointing it to that file. Also, the project uses `yarn` so we should use the `yarn` cache and not `npm`.
(also for the other workflows)

```yml
with:
  node-version-file: '.nvmrc'
  cache: 'yarn'
```

Reply:
cool, noted